.circleci/config.yml

```diff
@@ -34,7 +37,7 @@ jobs:
       - checkout
       - attach_workspace:
           at: *working-directory
-      - run: ./scripts/commit-lockfile.sh
+      - run: './scripts/exec-greenkeeper.sh greenkeeper-lockfile-upload'
```

Comment:
I think we should use [`npx`](https://www.npmjs.com/package/npx) here since it ships with `npm` and we can avoid needing a script all together:
```
npx greenkeeper-lockfile greenkeeper-lockfile-upload
```
I recommend this approach with a [different package](https://www.npmjs.com/package/framer-cli#-dependency-free) I built.